internal/declarative/v2/reconciler.go

```diff
@@ -615,23 +614,22 @@ func (r *Reconciler) configClient(ctx context.Context, obj Object) (Client, erro
 
 func (r *Reconciler) ssaStatus(ctx context.Context, obj client.Object,
 	requeueReason metrics.ManifestRequeueReason,
-	requeueType metrics.RequeueType,
 ) (ctrl.Result, error) {
 	resetNonPatchableField(obj)
-	r.Metrics.RecordRequeueReason(requeueReason, requeueType)
+	r.Metrics.RecordRequeueReason(requeueReason, queue.UnexpectedRequeue)
 	if err := r.Status().Patch(ctx, obj, client.Apply, client.ForceOwnership, r.FieldOwner); err != nil {
 		r.Event(obj, "Warning", "PatchStatus", err.Error())
 		return ctrl.Result{}, fmt.Errorf("failed to patch status: %w", err)
 	}
-	return ctrl.Result{Requeue: true}, nil
+
+	return ctrl.Result{RequeueAfter: r.RequeueIntervals.Busy}, nil
```

Comment:
To Reviewer:

basically if we need to update status, all requeueType here is unexpected, that's why I removed the requeueType parameter, and since all of them are unexpected requeue, there is no need to requeue immediately, in most case, this unexpected behavior can't be resolved in a short period, that's why we add a requeue interval to reduce the frequency.